framework/resources/Datasources/join/crt_tbl_prtntd_tbl.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/sh
+source conf/drillTestConfig.properties
+
+${DRILL_HOME}/bin/sqlline -n ${USERNAME} -p ${PASSWORD} -u "jdbc:drill:schema=dfs.Join;drillbit=${DRILL_STORAGE_PLUGIN_SERVER}"  --run=${DRILL_TEST_DATA_DIR}/Datasources/join/crt_tbls_partition_by_l.ddl
+${DRILL_HOME}/bin/sqlline -n ${USERNAME} -p ${PASSWORD} -u "jdbc:drill:schema=dfs.Join;drillbit=${DRILL_STORAGE_PLUGIN_SERVER}"  --run=${DRILL_TEST_DATA_DIR}/Datasources/join/crt_tbls_partition_by_r.ddl
+
+# it seems that sqlline does not exit, if one of the queries failed.
```

Comment:
The queries could fail for other reasons as well - not just absence of Sqlline. May be update the comment?